GroupIS1024/Models/Coach.cs

```diff
@@ -1,23 +1,29 @@
-﻿namespace GroupIS1024.Models
+﻿using Newtonsoft.Json;
+
+namespace GroupIS1024.Models
 {
     public class Coach
     {
+        [JsonProperty("Coach Id")]
         public int CoachId { get; set; }
 
+        [JsonProperty("First Name")]
         public string FirstName { get; set; }
 
+        [JsonProperty("Last Name")]
         public string LastName { get; set; }
 
-        public string Username { get; set; }
+        [JsonProperty("User Name")]
+        public string UserName { get; set; }
 
         public string State { get; set; }
 
-        public Boolean Memberstatus { get; set; }
-
-        public Team Team { get; set; }
+        [JsonProperty("Coach Status")]
```

Comment:
The variable name can be changed to Coach Status as it only belongs to Coach.

Reply:
You're already in a class named Coach, which gives context to the variable named Status.  So, I'd just leave it as Status, or describe what Status it is.